src/parsers/helper.ts

```diff
@@ -23,6 +23,18 @@ export const supportedFileType = (filePath: string): 'ts' | 'js' | undefined =>
   return undefined;
 };
 
+/**
+ * determine if the file is a react file(jsx, tsx);
+ * @param filepath
+ * @returns true or false
+ */
+export const isReactFileType = (filePath: string): boolean => {
```

Comment:
This is a misnomer. JSX is used for many other things than react files.

Reply:
removed the helper now, since it is not necessary when we now only checking against ts file.